eng/common/TestResources/build-test-resource-config.yml

```diff
@@ -0,0 +1,65 @@
+parameters:
+  - name: SubscriptionConfiguration
+    type: string
+    default: $(sub-config-azure-cloud-test-resources)
+  - name: SubscriptionConfigurations
+    type: object
+    default: null
+
+steps:
+  - ${{ if parameters.SubscriptionConfiguration }}:
+    - pwsh: |
```

Comment:
is this file supposed to be here?

Reply:
Oh snap.  I bet this is because I merged in master.  Let me remove it and make that update a separate PR.  Thanks!

Reply:
Thanks Dane.  Managing a feature branch is a bit of a dance in its own right.  Figuring it out....

Reply:
that it is haha. no worries